src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,19 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Checks if Person's tag list contains the original tag to be replaced, and replaces it if it exists.
+     *
+     * @param targetTag is the tag to be replaced
+     * @param newTag is the new tag to replace original tag
+     */
+    public boolean checkPersonTagList(Tag targetTag, Tag newTag) {
+        if (tags.contains(targetTag)) {
+            tags.replaceTag(targetTag, newTag);
+            return true;
+        }
+        else {
```

Comment:
Perhaps we can make this slightly more concise?

``` java

if (tags.contains(targetTag)) {
    blah blah...
    return true;
}
return false;
```

Would this be slightly better? =)